### PR TITLE
Fix scalar short price alert text

### PR DESCRIPTION
**Pull request by bthaile**
Need to convert price differently based on open order or trade.

image

https://github.com/AugurProject/augur/issues/7930

### CODE REVIEW

File: packages/augur-ui/src/modules/common/labels.tsx [hunk 1]
```diff
@@ -1089,7 +1089,7 @@ export const PendingLabel = (props: PendingLabelProps) => (
         </ReactTooltip>
       </>
     )}
-    {props.status && props.status === TXEventName.Failure && (
+    {status && status === TXEventName.Failure && (
```
Comment (bconfortin) [inline packages/augur-ui/src/modules/common/labels.tsx]: You don't need this since you've already destructured the props above:
`status && `

Comment (bconfortin) [reply packages/augur-ui/src/modules/common/labels.tsx]: You actually didn't need it before either

Comment (bthaile) [reply packages/augur-ui/src/modules/common/labels.tsx]: status can be null or undefined

Comment (bthaile) [reply packages/augur-ui/src/modules/common/labels.tsx]: but I see your point. I'll clean it up more

Comment (bthaile) [reply packages/augur-ui/src/modules/common/labels.tsx]: pushed, updated PR

**Review by bconfortin — APPROVED** [review]
Lgtm

**Pull request merged**